migrations/v2/V202409171800__v2.5.x_AIModel_TokenLimit.sql

```diff
@@ -0,0 +1,23 @@
+-- Insert a few new models into the AIModel table
+INSERT INTO [${flyway:defaultSchema}].[AIModel] ([Name], [Description], [Vendor], [AIModelTypeID], [PowerRank], [IsActive], [DriverClass], [DriverImportPath], [APIName], [SpeedRank], [CostRank], [InputTokenLimit]) VALUES ('Llama 3.1 405b', 'Llama 3.1 405 billion parameters', 'Groq', 'E8A5CCEC-6A37-EF11-86D4-000D3A4E707E', 5, 1, 'GroqLLM', NULL, 'llama-3.1-405b-reasoning', NULL, NULL, 128000);
+INSERT INTO [${flyway:defaultSchema}].[AIModel] ([Name], [Description], [Vendor], [AIModelTypeID], [PowerRank], [IsActive], [DriverClass], [DriverImportPath], [APIName], [SpeedRank], [CostRank], [InputTokenLimit]) VALUES ('Llama 3.1 70b', 'Llama 3.1 70 billion parameters', 'Groq', 'E8A5CCEC-6A37-EF11-86D4-000D3A4E707E', 4, 1, 'GroqLLM', NULL, 'llama-3.1-70b-versatile', NULL, NULL, 128000);
+INSERT INTO [${flyway:defaultSchema}].[AIModel] ([Name], [Description], [Vendor], [AIModelTypeID], [PowerRank], [IsActive], [DriverClass], [DriverImportPath], [APIName], [SpeedRank], [CostRank], [InputTokenLimit]) VALUES ('Llama 3.1 8b', 'Llama 3.1 8 billion parameters', 'Groq', 'E8A5CCEC-6A37-EF11-86D4-000D3A4E707E', 2, 1, 'GroqLLM', NULL, 'llama-3.1-8b-instant', NULL, NULL, 128000);
+INSERT INTO [${flyway:defaultSchema}].[AIModel] ([Name], [Description], [Vendor], [AIModelTypeID], [PowerRank], [IsActive], [DriverClass], [DriverImportPath], [APIName], [SpeedRank], [CostRank], [InputTokenLimit]) VALUES ('Claude 3.5 Sonnet', 'First model of the Claude 3.5 model family', 'Anthropic', 'E8A5CCEC-6A37-EF11-86D4-000D3A4E707E', 5, 1, 'AnthropicLLM', NULL, 'claude-3-5-sonnet-20240620', NULL, NULL, 200000);
+INSERT INTO [${flyway:defaultSchema}].[AIModel] ([Name], [Description], [Vendor], [AIModelTypeID], [PowerRank], [IsActive], [DriverClass], [DriverImportPath], [APIName], [SpeedRank], [CostRank], [InputTokenLimit]) VALUES ('GPT 4o Mini', 'Affordable and intelligent small model for fast, lightweight tasks. GPT-4o mini is cheaper and more capable than GPT-3.5 Turbo', 'OpenAI', 'E8A5CCEC-6A37-EF11-86D4-000D3A4E707E', 4, 1, 'OpenAILLM', NULL, 'gpt-4o-mini', NULL, NULL, 128000);
```

Comment:
These should also include `ID` in their VALUES– it's a uniqueidentifier so we can confidently set it here